app/helpers/sale_helper.rb

```diff
@@ -0,0 +1,9 @@
+module SaleHelper
+    def total_price
+        price = 0
+        @sales.each do |sale|
+            price += sale.book.price
+        end
+        price
+    end
```

Comment:
I think there is a better way to sum array of elements
maybe try [this](https://apidock.com/rails/Enumerable/sum)